src/components/funnels/funnels/register/steps/ticket/level.tsx

```diff
@@ -33,26 +34,27 @@ const AddonsContainer = styled.section`
 `
 
 const TicketLevel = (_: ReadonlyRouteComponentProps) => {
-	const { register, control, handleSubmit } = useFunnelForm<TicketLevelModel>(ChangeTicketLevel, SubmitTicketLevel)
-	const { ticketLevels, tshirtSizes, registrationExpirationDate } = useSiteMetadata()
+	const ticketType = useAppSelector(getTicketType())!
+	const { register, control, handleSubmit, watch } = useFunnelForm<TicketLevelModel>(ChangeTicketLevel, SubmitTicketLevel)
+	const level = watch('level', 'standard')
 
 	const { sizes, sizesByValue } = useMemo(() => {
-		const sizes = tshirtSizes.map(size => ({ value: size, label: size }))
+		const sizes = config.tshirtSizes.map(size => ({ value: size, label: size }))
```

Comment:
Probably OK, but might rename inner `sizes` to `shirtSizes` or something similar to prevent the scope clash